File: problems/0001_two_sum/v1_bruteforce/solution.ts
namespace BruteForce {
    function twoSum(nums: number[], target: number): number[] {
        for(let i=0; i<nums.length; i++){
            for(let j=i+1; j<nums.length; j++) {
                if (nums[i] + nums[j] == target){
                    return [i, j]
                }
            }
        }
        return [];
    }

    // Example usage:
    const nums = [3, 2, 4];
    const target = 6;
    const result = twoSum(nums, target);
    console.log(result); // Output: [1, 2]
}

File: problems/0001_two_sum/v1_bruteforce/solution.py
from typing import List

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        result = []
        for i in range(len(nums)):
            for j in range(i+1, len(nums)):
                print(nums[i], nums[j])
                if nums[i] + nums[j] == target:
                    result = [i, j]
        return result
    
# Example usage:
nums = [3,2,4]
target = 6
solution = Solution()
result = solution.twoSum(nums, target)
print(result)  # Output: [0, 1]